application-services/custom/camera-management/README.md

```diff
@@ -1,19 +1,117 @@
 # Camera Management Example App Service
-Camera management example Edgex application service to auto discover and connect to nearby ONVIF based cameras, 
-control cameras via commands, create inference pipelines for the camera video streams and publish inference
-results to MQTT broker.
+Use the Camera Management Example application service to auto discover and connect to nearby ONVIF and USB based cameras. This application will also control cameras via commands, create inference pipelines for the camera video streams and publish inference results to MQTT broker.
 
-This app uses [EdgeX Core Services][edgex-core-services], [Edgex Onvif device service][device-onvif-camera] and [Edge Video Analytics Microservice][evam].
+This app uses [EdgeX compose][edgex-compose], [Edgex Onvif device service][device-onvif-camera]], [Edgex Usb device service][device-usb-camera] and [Edge Video Analytics Microservice][evam].
 
 A brief video demonstration of building and using the example app service can be found [here](https://www.youtube.com/watch?v=vZqd3j2Zn2Y).
 
+## Install Dependencies
+
+### Environment
+This example has been tested with a relatively modern Lniux environment - Ubuntu 20.04 and later
+
+### Install Docker
+Install Docker from the official repository as documented on the [Docker](https://docs.docker.com/engine/install/ubuntu/) site.
+
+### Verify Docker
+To enable running Docker commands without the preface of sudo, add the user to the Docker group. Then run Docker with the `hello-world` test.
+
+1. Create Docker group:
+   ```bash
+   sudo groupadd docker
+   ```
+   >NOTE: If the group already exists, `groupadd` outputs a message: **groupadd: group `docker` already exists**. This is OK.
+
+2. Add User to group:
+   ```bash
+   sudo usermod -aG docker $USER
+   ```
+
+3. Refresh the group:
+   ```bash
+   newgrp docker 
+   ```
```

Comment:
I think we changed to recommending that the user reboot the system, as it is the "safer" option for people not as familiar with linux

Reply:
just say restart your computer for the changes to take effect